composer.json

```diff
@@ -32,13 +32,13 @@
         "symfony/asset": "^5.0",
         "symfony/yaml": "^5.0",
         "jms/translation-bundle": "^1.5",
-        "ezsystems/ezplatform-kernel": "^1.3@dev",
```

Comment:
I think those changes should be excluded from this PR.

Reply:
Yes, I have excluded them already. Your response was really fast. :)